### ISSUE TITLE
Implement Lisk

### ISSUE THREAD

**Issue by alepop** [trezor/trezor-wallet#240]
I would like to implement Lisk into the wallet. But I found in https://github.com/trezor/trezor-wallet/issues/151 that I should wait for `blockchain-link` will be published. Am I right?

**Comment by szymonlesisz** [comment]
Yes, we are currently working on a new module, an abstraction which will eventually replace [hd-wallet](https://github.com/trezor/hd-wallet/) and allow communication with various backend interfaces not only blockbook/bitcore.
As a proof of concept it will be implemented with [ripple-lib](https://github.com/ripple/ripple-lib/) first, then other interfaces will follow.

**Comment by alepop** [comment]
@szymonlesisz thanks for your response. any estimate about releasing the module?
